lib/sapience/grape.rb

```diff
@@ -11,6 +11,10 @@ class API
 
 module Sapience
   class Grape
+    UnsupportedVersion = Class.new(Exception)
```

Comment:
1. Because we don't need any extras and the ruby style guide prefers this way. 
2. Because it is exceptional meaning it isn't just a standard error because we don't support this. It should be reflected in the error.